archivist/constants.py

```diff
@@ -6,8 +6,10 @@
 ROOT = "archivist"
 SEP = "/"
 
-HEADERS_REQUEST_TOTAL_COUNT = "x-request-total-count"
-HEADERS_TOTAL_COUNT = "x-total-count"
+# define in MIME canonical form
+HEADERS_REQUEST_TOTAL_COUNT = "X-Request-Total-Count"
```

Comment:
Can we update these to match what apim will send?

Reply:
Well the Retry-After is specified and the code does not use anything else...
Are you referring to the Remaining-calls and Total-calls headers?

Reply:
after discussion set it to Archivist-Rate-Limit-Reset